### PR TITLE
FIx: test-unit version not compatible with latest spago on chapter 3 

### PR DESCRIPTION
**Pull request by MinsangKim-GreenLabs**
TLDR; this PR includes the dependency bump-up.

Hi, I'm new to the purescript world and trying some examples on the purescript-book website. Unfortunately, while I was installing these exercises, I got an issue that the version from a week ago couldn't execute on my machine. The problem was from chapter 3, as below.

```
❯ spago test
[1 of 7] Compiling Test.Unit.Console
Error found:
at .spago/test-unit/stackless-default/src/Test/Unit/Console.purs:1:1 - 32:50 (line 1, column 1 - line 32, column 50)

  A CommonJS foreign module implementation was provided for module Test.Unit.Console:

    .spago/test-unit/stackless-default/src/Test/Unit/Console.js

  CommonJS foreign modules are no longer supported. Use native JavaScript/ECMAScript module syntax instead.


See https://github.com/purescript/documentation/blob/master/errors/DeprecatedFFICommonJSModule.md for more information,
or to contribute content related to this error.


[error] Failed to build.
```

Luckily, I found a [discussion thread](https://discourse.purescript.org/t/after-upgrading-to-purs-spago-to-0-15-i-get-error-a-commonjs-foreign-module-implementation-was-provided/3001/4) talking about this issue, and a fixed version of the [equivalent repo](https://github.com/bodil/purescript-test-unit.git) (actually, this seems the original package that this repository supposed to have).

### CODE REVIEW

**Comment by JordanMartinez** [comment]
@MinsangKim-GreenLabs Thanks for the PR. I checked the [file on master](https://github.com/purescript-contrib/purescript-book/blob/master/exercises/chapter3/packages.dhall) and that file is correct. 

Based on the above error message, you are likely using PureScript 0.15 now, but the code you're trying to compile is PureScript 0.14 code. Perhaps you need to update your local copy of the book?

**Comment by JUSTIVE** [comment]
It seems that the master branch updated right after I forked it. Thanks for the comment; I'll close this PR
